README.md

```diff
@@ -0,0 +1,103 @@
+AzineEmailUpdateConfirmationBundle
+==================
+
+Symfony bundle which allows to use functionality of email change confirmation based on FOSUserBundle. 
+
+
+
+## Installation
+To install AzineEmailUpdateConfirmationBundle with Composer just add the following to your `composer.json` file:
+
+```
+// composer.json
+{
+    // ...
+    require: {
+        // ...
+        "azine/emailupdateconfirmation-bundle": "dev-master"
+    }
+}
+```
+Then, you can install the new dependencies by running Composer’s update command from 
+the directory where your `composer.json` file is located:
+
+```
+php composer.phar update
```

Comment:
many other projects now use `composer require azine/emailupdateconfirmation-bundle`, which updates the composer.json and installs the bundle. I think we should also document it like this.